test/support/test_config.h

```diff
@@ -16,6 +16,13 @@
 #ifndef _TEST_CONFIG_H
 #define _TEST_CONFIG_H
 
+// Any include from standard library required to have correct state of _GLIBCXX_RELEASE
+#if __cplusplus >= 202002L || _MSVC_LANG >= 202002L

```

Comment:
Better `#if __has_include(<version>)`; `__has_include` is a standard feature since C++17 so can be used freely.

See also https://github.com/oneapi-src/oneDPL/blob/5c8bb63e2d1288c4c2c71bc740ab1a4f80644fcd/test/support/test_macros.h#L19-L30 - I think it makes sense to eliminate duplicated code (but please, do not introduce yet another header just for these 5 lines) or at least have the same code if the headers should remain independent.

Reply:
Fixed